gradoop-flink/src/main/java/org/gradoop/flink/io/impl/dot/DOTDataSink.java

```diff
@@ -128,4 +149,28 @@ public void close() throws IOException {
       super.close();
     }
   }
+
+  /**
+   * Enumeration of supported dot formats.
+   */
+  public enum DotFormat {
+    /** Format that uses HTML tables to display element data. */
+    HTML,
+    /** Format that uses plain dot formatting. */
+    SIMPLE;
+
+    /**
+     * Returns a subclass of {@link AbstractDotFileFormat} that implements the specified formatting.
+     *
+     * @param printGraphHeadInformation flag that indicates that the graph head is included
+     * @return a subclass of AbstractDotFileFormat
+     */
+    public AbstractDotFileFormat getDotFileFormat(boolean printGraphHeadInformation) {
+      if (this == DotFormat.SIMPLE) {
+        return new DotFileFormatSimple(printGraphHeadInformation);
+      }
+      String textColor = "AAAAAAA";
```

Comment:
This looks like a magic value, please add a comment or name the variable to something more clear, like `colorGrey` or `htmlColorGrey`.